tests/epyccel/test_epyccel_return_arrays.py

```diff
@@ -0,0 +1,148 @@
+# pylint: disable=wrong-import-position, disable=missing-function-docstring, missing-module-docstring/
+import sys
+from numpy.random import randint, uniform
+import numpy as np
+import pytest
+
+sys.path.append('recognised_functions')
```

Comment:
Modifying the `sys.path` is hacky and dangerous. In addition you don't use the hack so I don't understand why you have done this